scripts/xnarost.lic

```diff
@@ -265,10 +265,10 @@ map_offset_y = nil
 narost_exit = false
 dragging = false
 
-if @theme_state == 'on' && File.exist?('maps-dark')
+if @theme_state && File.exist?('maps-dark')
```

Comment:
Should probably also account for older versions of Lich that still use on/off values. Otherwise could cause users who update to this version of xnarost but are still on old Lich to no longer be able to use dark-mode maps.